docs/advisories/cve_2017_5715.md

```diff
@@ -0,0 +1,78 @@
+## CVE-2017-5715
+
+| NAME         	| CVE-2017-5715                                                                                                                                                                                          	|
+|--------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Description  	| Systems with microprocessors utilizing speculative execution and branch prediction may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis. 	|
+| Source       	| https://nvd.nist.gov/vuln/detail/CVE-2017-5715                                                                                                              |
+| NVD Severity 	| medium (attack range: local) |
+| Last Updated  | Jan 01 2018 |
+
+##  Solutions
+
+This hardware exploit requires the installation of the Linux Kernel >= 4.4.110.
+kops is currently running the 4.4.x kernel.  A rolling-update or replacement of
+the kops ami image, with kops hosted on AWS is recommended. All platforms are
+affected not just AWS.
+
+### Impacted kops / kubernetes Components
+
+- kops ami
```

Comment:
Let's just say all AMIs without the newer kernel are vulnerable.  Likely all kernels built prior to 2018-01-03 are vulnerable.